Plugins/ServerPlugin.cpp

```diff
@@ -242,31 +242,86 @@ void CompilerClient::UpdateLoop() {
 ServerPlugin::ServerPlugin(MainWindow& mainWindow) : RGMPlugin(mainWindow) {
   // create a new child process for us to launch an emake server
   process = new QProcess(this);
+  
+  connect(process, &QProcess::errorOccurred, [=](QProcess::ProcessError error) { 
+    qDebug() << "QProcess error: " << error << endl; 
+  });
+  
+  #ifdef _WIN32
+  auto msys2Proc = new QProcess(this);
+
+  //TODO: Make all this stuff configurable in IDE
+  QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
+
+  // FIXME: this is just an approximate guess on how to get emake running outside a msys shell and currently causes emake to crash
+  QString msysPath;
+  if (!env.contains("MSYS_ROOT")) {
+    msysPath = env.value("SystemDrive", "C:") + "/msys64/";
+    qDebug() << "Enviornmental variable \"MSYS_ROOT\" is not set defaulting MSYS path to: " + msysPath;

```

Comment:
This has double quotes because of QString, see the Qt Forum post for explanation.
https://forum.qt.io/topic/71313/double-quotes-in-qstring/7

Workaround is use `std::string`, do `QString(...).toUtf8().constData()` on the QString, `qDebug().noquote()`, or finally use some other means of output.

Reply:
I just did 289d739041de948894d26f9c19159b6f68b7b3be and went ahead and fixed it.